Gemfile.lock

```diff
@@ -231,7 +231,9 @@ GEM
     middleman-syntax (3.2.0)
       middleman-core (>= 3.2)
       rouge (~> 3.2)
-    mimemagic (0.3.5)
+    mimemagic (0.3.9)
```

Comment:
```
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /Users/christopherashton/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/mimemagic-0.3.9/ext/mimemagic
/Users/christopherashton/.rbenv/versions/2.7.2/bin/ruby -I/Users/christopherashton/.rbenv/versions/2.7.2/lib/ruby/2.7.0/rubygems -rrubygems
/Users/christopherashton/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rake-13.0.3/exe/rake
RUBYARCHDIR\=/Users/christopherashton/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/extensions/x86_64-darwin-18/2.7.0/mimemagic-0.3.9
RUBYLIBDIR\=/Users/christopherashton/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/extensions/x86_64-darwin-18/2.7.0/mimemagic-0.3.9
rake aborted!
Could not find MIME type database in the following locations: ["/usr/local/share/mime/packages/freedesktop.org.xml",
"/opt/homebrew/share/mime/packages/freedesktop.org.xml", "/usr/share/mime/packages/freedesktop.org.xml"]

Ensure you have either installed the shared-mime-info package for your distribution, or
obtain a version of freedesktop.org.xml and set FREEDESKTOP_MIME_TYPES_PATH to the location
of that file.

Tasks: TOP => default
(See full trace by running task with --trace)

rake failed, exit code 1
```

Reply:
Documented my local fix in a tacked-on commit.

Provided the build passes, this should be ready to go 👍